src/loaders/thematicLoader.js

```diff
@@ -180,26 +184,21 @@ const thematicLoader = async (config) => {
     if (!valueFeatures.length) {
         if (!features.length) {
             alerts.push({
-                warning: true,
-                message: i18n.t('Selected org units: No coordinates found', {
-                    nsSeparator: ';',
-                }),
+                code: WARNING_NO_OU_COORD,
+                message: i18n.t('Thematic layer'),
             })
         } else {
             alerts.push({
-                warning: true,
-                message: `${name}: ${i18n.t('No data found')}`,
+                code: INFO_NO_DATA,
+                message: name,
             })
         }
     }
 
     if (coordinateField && !associatedGeometries.length) {
```

Comment:
Similar to EarthEngineLoader.js L65, we get alerts for both OU missing geom and missing associated geom.

![image](https://github.com/user-attachments/assets/b04d53af-9094-43e5-bc1d-d4e1017ba773)